### ISSUE TITLE
[Bug]: 400 bad request when logging to trakt with 6.0.89

### ISSUE THREAD

**Issue by WhyRib** [nova-video-player/aos-AVP#784]
### Problem description

400 bad request appears after inputting my credentials then confirming that i want to link Nova with Trakt and login fails.

### Steps to reproduce the issue

When trying to log to trakt, input my credentials, then check yes when invited to link Nova with trakt.

### Expected behavior

Log in and start syncing with trakt.

### Your phone/tablet/androidTV model

Galaxy Tab S7

### Operating system version

Android 13

### Application version and app store

6.0.89 from github

**Comment by courville** [comment]
Cf. https://github.com/nova-video-player/aos-AVP/issues/723
Are you using an adblocker?

**Comment by WhyRib** [comment]
I thought I did not ... Then reading your answer I remembered that my internet provider has an ad-blocker by default in the router settings (Free, for anyone in France who may have the same issue).

Got confused by the fact that when using adguard, you do not even reach the trakt login window.

Thank you very much for your reply and sorry for wasting your time. (Oh, and thank you for this great app !).
